extensions/lifecycle/LifecycleQueuePopulator.js

```diff
@@ -422,48 +422,60 @@ class LifecycleQueuePopulator extends QueuePopulatorExtension {
             return;
         }
 
-        this.log.trace(
-            'publishing object delete entry',
-            { bucket: entry.bucket, key: entry.key, version: value.versionId },
-        );
-
-        const topic = `${coldStorageGCTopicPrefix}${locationName}`;
-        const key = `${entry.bucket}/${entry.key}`;
+        const ownerId = value['owner-id'];
+        this.vaultClientWrapper.getAccountId(ownerId, (err, accountId) => {
```

Comment:
i don't think we can/want to afford the extra call to Vault for each message, when the value is actually not used?